README.md

```diff
@@ -30,6 +30,23 @@ Currently developed with Ruby 2.4.0. Written for Ruby 2.3 and up.
 
 ## Getting started
 
+Add to your Gemfile
+```
+gem 'kraken_ruby_client', github: 'jonatack/kraken_ruby_client'
+```
+
+Install
+
+```
+bundle isntall
```

Comment:
```suggestion
bundle install
```

Reply:
Thanks @sirwolfgang for reviewing. Change done in https://github.com/jonatack/kraken_ruby_client/pull/13.